server.go

```diff
@@ -28,6 +28,14 @@ type contextVal struct {
 	actorName string
 }
 
+// Most underlying usage of grid.Server's only needs to
+// be able to create a grid.Mailbox so this provides a
+// way for users to implement test servers without spinning
+// up a whole grpc server/client and etcd client/watch
+type MailboxServer interface {
+	NewMailbox(name string, size int)
+}
+
```

Comment:
I don't think we need to define this interface here; the things that use Grid (Linkgrid, etc.) can define it locally.